README.md

```diff
@@ -18,6 +18,7 @@
 5. have bot listing requirements (such as no NSFW outside NSFW channels, respect rate-limits, don't respond to other bots, etc.)
 6. blur NSFW profile pictures.
 7. have a ToS and Privacy page.
+8. not be a copy of other's bot lists (such as vcodes, sank6, RDL, and others that are open sourced, including lists that are edited or modified from other's lists).
```

Comment:
Not good. BotBlock doesn't deny bot lists that use a template.
This is at most a *should* and not a *must*.

Reply:
if botblock didnt deny anyone till now, they should start denying. I cant see why they should be approved for a list which they never made it in the first place

Reply:
+1, this is potentially something to include in the _should_ list, but not the must. Also, please do not call out specific examples like this, we're not here to disparage certain projects.

Reply:
um k

Reply:
fixed it in latest commit